back/scripts/utils/geolocator.py

```diff
@@ -97,16 +97,19 @@ def add_geocoordinates(self, frame: pd.DataFrame) -> pd.DataFrame:
             how="left",
         )
 
+        to_concat = [reg_dep_ctu]
         epci = frame[~frame["type"].isin(["REG", "DEP", "CTU", "COM"])].merge(
             self._get_epci_coords(),
             on=["type", "siren"],
             how="left",
         )
-
+        to_concat.append(epci)
         cities = frame[frame["type"] == "COM"]
-        geolocator_response = self._request_geolocator_api(
-            cities[["code_insee", "nom"]].drop_duplicates()
-        )
-        cities = cities.merge(geolocator_response, on=["type", "code_insee"], how="left")
 
-        return pd.concat([reg_dep_ctu, epci, cities])
+        if not cities.empty:
```

Comment:
remark : while the fix is probably good, we should find why cities was empty in the first place.

Reply:
C'est bien vu, je suis justement en train de revoir complètement cette partie car elle donne très peu de résultats. Mieux vaut la laisser inchangée, si ça ne te dérange pas, pour éviter des conflits.